core/sandbox/package.json

```diff
@@ -17,23 +17,22 @@
     "@dotcom-tool-kit/circleci-heroku": "file:../../plugins/circleci-heroku",
     "@dotcom-tool-kit/eslint": "file:../../plugins/eslint",
     "@dotcom-tool-kit/frontend-app": "file:../../plugins/frontend-app",
+    "@dotcom-tool-kit/jest": "file:../../plugins/jest",
     "@dotcom-tool-kit/lint-staged": "file:../../plugins/lint-staged",
     "@dotcom-tool-kit/lint-staged-npm": "file:../../plugins/lint-staged-npm",
     "@dotcom-tool-kit/mocha": "file:../../plugins/mocha",
-    "@dotcom-tool-kit/jest": "file:../../plugins/jest",
     "@dotcom-tool-kit/n-test": "file:../../plugins/n-test",
+    "@dotcom-tool-kit/next-router": "file:../../plugins/next-router",
+    "@dotcom-tool-kit/nodemon": "file:../../plugins/nodemon",
     "@dotcom-tool-kit/npm": "file:../../plugins/npm",
     "@dotcom-tool-kit/prettier": "file:../../plugins/prettier",
     "@dotcom-tool-kit/upload-assets-to-s3": "file:../../plugins/upload-assets-to-s3",
-    "dotcom-tool-kit": "file:../cli"
+    "dotcom-tool-kit": "file:../cli",
+    "nodemon": "^2.0.15"
```

Comment:
nitpick: nodemon is a peerdep in the plugin, so we can remove this

Reply:
Even though peer dependencies will be installed automatically I think it's still a good idea to manually specify the version in the end product if we are explicitly using it – like we are in our `package.json`'s `"scripts"` field – so that the major version won't suddenly change if the peer dependency's version range is widened.